Content.Server/Medical/BiomassReclaimer/BiomassReclaimerSystem.cs

```diff
@@ -127,6 +128,17 @@ private void OnShutdown(EntityUid uid, ActiveBiomassReclaimerComponent component
             _ambientSoundSystem.SetAmbience(uid, false);
         }
 
+        private void OnPowerChanged(EntityUid uid, BiomassReclaimerComponent component, ref PowerChangedEvent args)
+        {
+            if (args.Powered)
+            {
+                if (component.ProcessingTimer > 0)
+                    AddComp<ActiveBiomassReclaimerComponent>(component.Owner);
```

Comment:
this should be ensurecomp because this event is also raised when changing providers (e.g. the cable we were using got snipped but there's another in range) and addcomp will error if it already has the component